plasmapy/particles/particle_class.py

```diff
@@ -1685,6 +1776,27 @@ def __repr__(self):
         """
         return f"DimensionlessParticle(mass={self.mass}, charge={self.charge})"
 
+    def json_dict(self) -> dict:
```

Comment:
If the superclass has `json_dict` as a property, shouldn't this also be a property?

Reply:
Ah, that's right. Thank you for pointing it out!

Reply:
```suggestion
    @property
    def json_dict(self) -> dict: 
```
halping! :laughing: